styles/layout/_styles.scss

```diff
@@ -59,3 +59,38 @@
 .right-justify {
     text-align: right;
 }
+
+.center {
+    margin: 0 auto;
+}
+
+.copyright {
```

Comment:
We thought about that but since it's not a logo we decided for layout. On the other hand it's used to identify something. I think both are valid. @joebri ?